### PR TITLE
Pre-Auth Registration Approval

### PR DESCRIPTION
**Pull request by nadinequinr3**
Preview available here: https://nadine.preview.docs.r3.com/en/platform/corda/5.0-beta/operating/registration-approval.html

### CODE REVIEW

File: content/en/platform/corda/5.0-beta/operating/operating-tutorials/registration-requests/configuring-manual-approval-rules.md
Comment (nadinequinr3) [inline content/en/platform/corda/5.0-beta/operating/operating-tutorials/registration-requests/configuring-manual-approval-rules.md:1]: This content was already reviewed as part of https://github.com/corda/corda-docs-portal/pull/1175 but the file was moved to a new folder in this PR

**Review by yarivus — APPROVED** [review]
LGTM

File: content/en/platform/corda/5.0-beta/operating/operating-tutorials/registration-requests/preauthenticating-tokens.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    "status": "REVOKED",
+    "ttl": "2022-06-24T10:15:30Z"
+  }
+```
+
+You can also pass the following optional properties to filter or expand the search results:
+* ownerX500Name — the X.500 name of the member who the token was issued for. 
+This is passed as a URL query parameter with the full URL encoded X.500 name.
+* preauthtokenid — the ID of a specific token to look up.
+* viewinactive — set this to `true` to inlude consumed, revoked, or expired tokens.
```
Comment (charlieR3) [inline content/en/platform/corda/5.0-beta/operating/operating-tutorials/registration-requests/preauthenticating-tokens.md:58]:
```suggestion
* viewinactive — set this to `true` to include consumed, revoked, or expired tokens.
```

**Review by charlieR3 — APPROVED** [review]
LGTM

File: content/en/platform/corda/5.0-beta/operating/registration-approval.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+## Pre-Authentication
+
+The network operator can pre-authenticate registrations for specific members. This allows a registering member to bypass approval rules defined for the group. Authentication is done outside of Corda using any criteria the operator chooses. Once the network operator has completed their authentication process, they can generate a one-time-use pre-authentication token, also known as a pre-auth token, specific to the authenticated member.
+
+Corda has a set of REST APIs available for managing these pre-auth tokens. Through these APIs, tokens can be created, revoked, and viewed. When viewing a token, it is possible to see the token ID, the X.500 name of the member the token is assigned to, optionally a time and date when the token expires, the token status, and additional information provided by the MGM when creating or revoking the token. For more information, see [Managing Pre-Authentication Tokens]({{< relref "./operating-tutorials/registration-requests/preauthenticating-tokens.md" >}})
```
Comment (AnnaK-R3) [inline content/en/platform/corda/5.0-beta/operating/registration-approval.md:35]:
```suggestion
Corda has a set of REST APIs available for managing these pre-auth tokens. Through these APIs, tokens can be created, revoked, and viewed. When viewing a token, it is possible to see the token ID, the X.500 name of the member the token is assigned to, optionally a time and date when the token expires, the token status, and additional information provided by the MGM when creating or revoking the token. For more information, see [Managing Pre-Authentication Tokens]({{< relref "./operating-tutorials/registration-requests/preauthenticating-tokens.md" >}}).
```

**Review by yarivus — DISMISSED** [review]
LGTM

**Review by AnnaK-R3 — APPROVED** [review]

**Review by yarivus — COMMENTED** [review]
thx @nadinequinr3

**Pull request merged**